### DISCUSSION TITLE
weather.com ads

### DISCUSSION THREAD

**Discussion by cmcjsrb** [uBlockOrigin/uAssets#15076] [General]
weather.com ads showing again--have done purge of cache--update--they still show--once again can you help?

**Comment by gorhill** [comment]
Not seeing ads here. Use the chat icon in the popup panel to report such issues, we need far more information than what you provide here. The chat icon will fill in all this information in a formal filter issue.

**Comment by mapx-** [comment]
@cmcjsrb continue here: https://github.com/uBlockOrigin/uAssets/issues/15077
